beheer.md

```diff
@@ -104,13 +104,13 @@ docker service ps getstartedlab_web | Haal de gegevens op van alle tasks die de
 docker stack rm getstartedlab | Afsluiten van de applicatie.
 docker swarm leave --force | Afsluiten van de swarm.
 
-**Criteria**
-
-* API specificatie (OAS3) moet beschikbaar zijn en uit de imvertor genereerbaar zijn
-* Documentatie (zowel technisch als functioneel) moet beschikbaar zijn
-* De referentie-implementatie moet gereed en up-to date zijn met dan geldende OAS3
-* Het gebruikte Informatiemodel (minimaal UML)
-* Geautomatiseerde test en buildstraat (CI) moet beschikbaar voor de RI
-* De standaard moet ingebouwd zijn bij leveranciers (3 of meerdere leveranciers)
-* De standaard bevat een tabel waarin wordt aangegeven welke API versies met welke andere API versies compatible zijn.
+##Criteria##
+
+- API specificatie (OAS3) moet beschikbaar zijn en uit de imvertor genereerbaar zijn
+- Documentatie (zowel technisch als functioneel) moet beschikbaar zijn
+- De referentie-implementatie moet gereed en up-to date zijn met dan geldende OAS3
+- Het gebruikte Informatiemodel (minimaal UML)
+- Geautomatiseerde test en buildstraat (CI) moet beschikbaar voor de RI
+- De standaard moet ingebouwd zijn bij leveranciers (3 of meerdere leveranciers)
+- De standaard bevat een tabel waarin wordt aangegeven welke API versies met welke andere API versies compatible zijn.
```

Comment:
Dat is toch af te leiden uit het versienummer? Als ik Joeri goed begrepen heb zijn de minors compatible want alleen maar non-breaking changes. Heeft een versie een major andere versie dan is er sprake van een breaking change en zijn de versies niet compatible.

Reply:
@michielverhoef Volgens mij was dit een wens van @joeribekker en het ging er niet om de backwardse compatibility ut te drukken maar de compatibilteit met andere API's. Stel bijv. dat een van de ZGW API's gebruik maakt van de BIP API. Met welke versie van die API is hij dan compatible.

Reply:
Yes.

Reply:
Ah ok. Dan zou ik er iets van maken als "welke versies van een API met welke versies van een andere API compatible zijn."
Dat lijkt mij iets duidelijker. Ik dacht nu dat het over welke versies van dezelfde API compatible met elkaar waren.

Reply:
Volgens mij is dit nu wel duidelijk en duidelijker geformuleerd.